sample/README.md

```diff
@@ -0,0 +1,43 @@
+picoquic sample
+===============
+
+The sample program is a simple QUIC client/server demo.
+
+Building
+--------
+picoquic\_sample is built as part of the compilation process of picoquic. It is
+will be available in the root folder.
+
+Usage
+-----
+Usage:
+    ../picoquic_sample client server_name port folder *queried_file
+or :
+    ../picoquic_sample server port cert_file private_key_file folder
+
+Example
+-------
+
+Generate the certificates:
+```
+openssl req -x509 -newkey rsa:4096 -days 365 -keyout ca-key.pem -out ca-cert.pem
+openssl req -newkey rsa:4096 -keyout server-key.pem -out server-req.pem
+```
+These commands will prompt a few questions, you don't need to put actual data
```

Comment:
RSA 4096 will make the initial exchange kind of slow. Using ECDSA is a bit more complex, so I get sticking with RSA for the sample is fine, but 4096 is probably overkill. 2048 would be adequate.